packages/pwa-buildpack/lib/WebpackTools/plugins/__tests__/__fixtures__/basic-project-1-page/robots.txt

```diff
@@ -0,0 +1 @@
+Deny * from all
```

Comment:
Does this PR need to be rebased? I feel like I've seen this file go by before.

Reply:
Oh possible